creator/lib/screens/salon_application_screen.dart

```diff
@@ -218,7 +221,7 @@ class _SalonApplicationScreenState extends State<SalonApplicationScreen> {
             top: 16,
           ),
           child: Text(
-            FirebaseAuth.instance.currentUser.email,
+            FirebaseAuth.instance.currentUser?.email ?? "",
```

Comment:
```suggestion
            FirebaseAuth.instance.currentUser?.email ?? '',
```

Please use single quotes when not using string interpolation

Reply:
Thanks, I'll fix it.